### PR TITLE
"Create" function name search heuristic

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: test/vmtester/vmtester.cpp [hunk 1]
```diff
@@ -35,6 +35,38 @@ evmc_instance* get_vm_instance()
     return vm.get();
 }
 
+std::vector<std::string> get_vm_names(const fs::path path)
```
Comment (axic) [inline test/vmtester/vmtester.cpp]: Why not make this into a helper in the library (like instructions)? It has to have the filename as an input string though, so in the `vmtester` you could have this extra function to deal with `fs::path`.

Comment (chfast) [reply test/vmtester/vmtester.cpp]: Maybe I will do it later to use in aleth. But not useful for geth anyway.

File: include/evmc/evmc.h [hunk 2]
```diff
@@ -732,16 +732,19 @@ struct evmc_instance
...
+#if EVMC_DOCUMENTATION
 /**
  * Example of a function creating an instance of an example EVM implementation.
  *
- *  Each EVM implementation MUST provide a function returning an EVM instance.
- *  The function SHOULD be named `evmc_create_<vm-name>(void)`.
+ * Each EVM implementation MUST provide a function returning an EVM instance.
+ * The function SHOULD be named `evmc_create_<vm-name>(void)`. If the VM name contains hyphens
+ * replaces them with underscores in the function names. For VMs distributed as shared libraries,
+ * the name of the library SHOULD match the VM name.
```
Comment (axic) [inline include/evmc/evmc.h]: `libhera` -> `evmc_create_libhera` ?

Comment (chfast) [reply include/evmc/evmc.h]: The "lib" prefix is skipped if present.

Comment (chfast) [reply include/evmc/evmc.h]: Done.

**Pull request merged**